README.md

```diff
@@ -184,62 +257,89 @@ Online Shop:
 >
 > Add a new "Checkout" button on the shopping cart page, which creates a new order on the backend.
 
-Further Resources:
+#### Further Resources (Optional):
 
 - [CORS: Proxy-ing a backend server](https://angular.io/guide/build#proxying-to-a-backend-server)
 - [Http error handling](https://angular.io/guide/http#error-handling)
+- [Unsubscribe from RxJs Observables](https://www.thisdot.co/blog/best-practices-for-managing-rxjs-subscriptions)
```

Comment:
Might be worth suggesting somewhere to create an abstract class that can be inherited to deal with this.

Reply:
Good point

Reply:
Done